Scroll restoration test fails execution on TeamCity

```
java.lang.AssertionError: Invalid scrollX, expected 1600 actual 1584
    at org.junit.Assert.fail(Assert.java:88)
    at com.vaadin.flow.uitest.ui.scroll.ScrollIT.verifyScrollX(ScrollIT.java:341)
```

--- a/flow-tests/test-root-context/src/test/java/com/vaadin/flow/uitest/ui/scroll/ScrollIT.java
+++ b/flow-tests/test-root-context/src/test/java/com/vaadin/flow/uitest/ui/scroll/ScrollIT.java
@@ -9,8 +9,8 @@ import com.vaadin.testbench.By;
 
 public class ScrollIT extends ChromeBrowserTest {
 
-    // Scroll position may differ a little locally and in Travis
-    private static final int allowedScrollVariance = 15;
+    // Scroll position may differ a little locally and in Travis and Test grid
+    private static final int allowedScrollVariance = 20;
 
     @Test
     public void testScrollRestoration_basicBackForward() {

